time stamp

float64 x
float64 y
float64 z

float64 x_raw
float64 y_raw
float64 z_raw

string byte0
string byte1
string byte2
string byte3
string byte4
string byte5
